### PR TITLE
feat: support summary to msteams notification

### PR DESCRIPTION
**Pull request by dongjiang1989**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

support `summary` to msteams notification. 

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [x] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [Prometheus-Operator testing guidelines](../TESTING.md) for recommendations about automated tests.

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
support summary to msteams notification
```

### CODE REVIEW

**Comment by slashpai** [comment]
There is already a PR for this https://github.com/prometheus-operator/prometheus-operator/pull/6206

File: test/framework/framework.go [hunk 1]
```diff
@@ -83,6 +88,14 @@ func New(kubeconfig, opImage, exampleDir, resourcesDir string, operatorVersion s
 		return nil, fmt.Errorf("build config from flags failed: %w", err)
 	}
 
+	if config.QPS < DefaultQPS {
```
Comment (dongjiang1989) [inline test/framework/framework.go:91]:
```suggestion
	// setting DefaultQPS and DefaultBurst
	if config.QPS < DefaultQPS {
```

**Comment by dongjiang1989** [comment]
> There is already a PR for this #6206

Thanks. Got it.

File: test/framework/framework.go [hunk 2]
```diff
@@ -59,6 +59,11 @@ const (
 	operatorTLSDir = "/etc/tls/private"
 )
 
+var (
+	DefaultQPS   = float32(20)
+	DefaultBurst = int(40)
```
Comment (dongjiang1989) [inline test/framework/framework.go:64]:
```suggestion
	DefaultQPS   = float32(30)
	DefaultBurst = int(60)
```

**Comment by ArthurSens** [comment]
Closing this one since #6206 was merged! Thanks anyway for the PR!

**Pull request closed without merging**